pipeline-runner/src/init.r

```diff
@@ -274,6 +288,9 @@ init <- function() {
                     error = error_txt,
                     cause = cause
                 )
+
+                send_pipeline_fail_update(pipeline_config, input_parse$experimentId, input_parse$processName, "Error message placeholder")
```

Comment:
No, because that involves more work and refactoring in the UI, the api and the pipeline that is not ideal to be done at this stage.

The main focus of this PR was fixing qc and gem2s already existing error management and at the same time bringing gem2s structure a little bit closer to qc's.